packages/asset-emitter/README.md

```diff
@@ -0,0 +1 @@
+# @typespec/asset-emitter
```

Comment:
Ooph - should we link to the current website docs page or trying to dissuade usage?

Reply:
Yeah not sure yet, with the intermediate stage we are at, need to have this rewrite of the library author docs so might leave it for that